### DISCUSSION TITLE
How to remove "more" link and wrap text in list view

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#16229] [How to]
I am bit confused with steps in removing "more" and wrapping text in list view as per screenshot. Any help please?
![image](https://user-images.githubusercontent.com/84110093/177032253-cc92c696-0fb2-450e-9f92-63a0109084cf.png)

**Comment by mariuszkrzaczkowski** [comment]
set a larger value in the variable: https://doc.yetiforce.com/code/classes/Config-Main.html#property_listview_max_textlength

**Comment by rskrzypczak** [comment]
.. or https://github.com/YetiForceCompany/YetiForceCRM/commit/0653a500ef053c89cc0f4ee2e6c65ba2a43199de
![image](https://user-images.githubusercontent.com/10221999/177096434-0f5606ad-24d9-43dc-a11f-4f02ac153d72.png)

**Comment by s2b-git** [comment]
Thanks guys. Unfortunately this doesn't apply to custom module. 
All other modules inheriting listview_max_textlength number of chars but custom still short
